bin/scripts/name_parser/name_parser_test1

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```

Comment:
getting an error running the script:
```
  File "name_parser_test1", line 150
    print(o1, o2, sep='\n')
                     ^
SyntaxError: invalid syntax
```

This seems to be because my version of fontforge is using python 2. Whereas `font-patcher` works fine because of `from __future__ import print_function`

this is likely because my `fontforge` is pretty old...

Reply:
I believe no-one has python2 but not python3 anymore, given the python2 is EOL since 2020.

I used a non-specific shebang in the scripts, that ended up with python3 on machines like mine where there is no python2 anymore.

Changed the shebang to specifically reference python3.

That should fix it on your machine? We could of course maintain some python2 compatibility, but for YEARS I hoped that we could finally drop all that legacy. As these are stand-alone scripts I guess it is ok to add a python3 dependency?